ex02.ex

```diff
@@ -53,7 +53,10 @@ defmodule Ex02 do
   # if the first two elements of a list are equal                #
   ################################################################
 
-  first2equal = your_anonymous_function([])
+  first2equal = fn 
```

Comment:
## -4

Although this passes the test, it only works for lists of length 5. I was expecting something like:

~~~ elixir
  first2equal = fn
                  [ a, a | _ ]              -> true
                  other when is_list(other) -> false
                end
~~~

or

~~~elixir
  first2equal = fn
                  [ a, b | _ ]              -> a == b
                end
~~~